### DISCUSSION TITLE
Support for "Bar" format larger screens - Mipi or SPI

### DISCUSSION THREAD

**Discussion by b1gmans** [Bodmer/TFT_eSPI#1868] [Compatible displays and setup files]
Hi, I have seen these larger screens:

https://www.buydisplay.com/spi-1280x400-7-84-ips-tft-lcd-module-with-ra8876-capacitive-touch-panel

https://www.buydisplay.com/4-6-inch-color-bar-tft-lcd-iot-display-800x320-pixels-with-optl-touch-screen

looks like first one has a SSD2828 Controller added,  so can take SPI input, (also the bare panel available with driver EK79030AC7,MIPI interface.)  second one is Mipi - ILI6122 driver

Any plans to support these larger, bar format displays in future please?

**Comment by Bodmer** [comment]
I can only support displays that I have bought myself and tested. The SSD1963 is the largest supported display.
